src/patterns/error-validation/examples/errors-proto/index.njk

```diff
@@ -224,6 +228,32 @@ layout: none
                 }
             })
         }}
+
+        {{
+            onsDuration({
+                "label": "Duration",
+                "id": "address-duration",
+                "dontWrap": true,
+                "field1": {
+                    "id": "address-duration-hours",
+                    "name": "address-duration-hours",
+                    "label": "Hours",
+                    "attributes": {
+                        "min": 0,
+                        "max": 100
```

Comment:
We said we'd remove these attributes for durations because they don't work on the text type input